lib/cosmos/api/cmd_api.rb

```diff
@@ -181,7 +181,7 @@ def cmd_raw_no_checks(*args, scope: $cosmos_scope, token: $cosmos_token, **kwarg
     # @param data [String] The raw binary data
     def send_raw(interface_name, data, scope: $cosmos_scope, token: $cosmos_token)
       authorize(permission: 'cmd_raw', interface_name: interface_name, scope: scope, token: token)
-      Store.instance.write_interface(interface_name, { 'raw' => data }, scope: scope)
+      raise 'unimplemented'
```

Comment:
This still needs work so I marked it as such since there is no write_interface store method

Reply:
We do need to make this work.

Reply:
Will address in another PR.